spec/features/recipes/show_recipe_page_spec.rb

```diff
@@ -0,0 +1,21 @@
+feature 'user visits show recipe page' do
+  before do
+    FactoryBot.create(:recipe, :scrambled_eggs)
+    FactoryBot.create(:recipe, :tikka_masala)
+    visit('/')
+    page.all(:xpath, '//a[contains(text(), "Show")]')[1].click
+  end
+
+  it 'shows user clicked recipe' do
+    expect(page).to have_current_path(%r{recipes/\d+})
+
+    expect(page.body).to match(%r{img.*src="\/assets\/recipes-.+\.png"})
+    expect(page).to have_content 'tikka masala'
+    expect(page).to have_content 'tikka a masal'
+    expect(page).to have_content 'tikka 100 mililiter'
+    expect(page).to have_content 'masala 10 gram'
+
+    click_on('Back')
+    expect(page).to have_current_path('/recipes')
```

Comment:
The description of the ticket does not mention anything about `Back` button feature. Most browsers (if not all) have builtin back button so no need for such fancy things. Please remove the button (and the test for it).